src/Main.java

```diff
@@ -2,5 +2,40 @@
 
     public static void main(String[] args) {
         // Please code here
+        int c=1;
+        for(int i = 10; i >=1; i--)             //No. of column of upper half of triangle
```

Comment:
Input the maximum number of stars in a row from user and then change calculations accordingly

Reply:
Task is updated, Please review.